curriculum/challenges/english/25-front-end-development/lab-bank-account-manager/6718d2d59337c822ecb697ff.md

```diff
@@ -0,0 +1,275 @@
+---
+id: 6718d2d59337c822ecb697ff
+title: Build a Bank Account Management Program
+challengeType: 14
+dashedName: build-a-bank-account-management-program
+---
+
+# --description--
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab. 
+
+**User Stories:**
+
+1. You should define a class named `BankAccount` with a constructor that takes `balance` as parameter. The constructor should set the default `balance` to `0`and initialize an empty array named `transactions` to store transaction records as objects. 
```

Comment:
```suggestion
1. You should define a class named `BankAccount` with a constructor that takes `balance` as a parameter. The constructor should set the `balance` to `0` if no value is provided and initialize an empty array named `transactions` to store transaction records as objects.
```

Reply:
The constructor would always set balance to `0` and doesn't accept other values.